lib/content-store/src/store/pg/migrations/U0001__setup_primitives.sql

```diff
@@ -0,0 +1,69 @@
+-- TODO: add constraints, plpgsql doesn't do domain constraints though:
```

Comment:
Do we need this migration? I'm not sure where/why we'd be creating ULIDs from within the DB for the content store.

Reply:
Hmmm... good point. We likely don't. I'll run through it again, but I think I originally wanted something from there. Now, I don't think so.